src/main/java/frc/robot/commands/DriveCommand.java

```diff
@@ -67,13 +67,13 @@ public enum TargetMode {
 
   private final SwerveRequest.FieldCentric fieldCentricDrive =
       new SwerveRequest.FieldCentric()
-          .withDeadband(maxSpeed * 0.1)
-          .withRotationalDeadband(maxAngularRate * 0.1) // Add a 10% deadband
+          .withDeadband(maxSpeed * 0.01)
```

Comment:
Let's try driving with zero for these deadband values.   I think it's not needed anymore, since we have a deadband on the controls already